examples/kafka_example.yaml

```
@@ -24,6 +24,7 @@ streams:
       type: kafka
       brokers:
         - localhost:9092
-      topic: test-topic-copy
+      topic: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)